File: Praca/spr.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  spr.py
#  
#  Copyright 2018  <>





def main(args):
    n = int(input("Podaj liczbe:"))
    i = 2;
    while i * i <= n:
        if n % i == 0:
            print("Zlozone")
            return 0
         i += 1       
        print("Pierwsza")
        return 0
    
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: Praca/spr.cxx
/*
 * spr.cxx
 * 
 * Copyright 2018  <>
 * 

 * 
 */


#include <iostream>
using namespace std;


int main(int argc, char **argv)
{
    
    int n = 0;
    cout << "Podaj liczbe";
    cin >> n;
    int i =2;
    for(i = 2; i*i <= n; i++) {
        
        
        
        
        if(n % i == 0)  {
            
            cout << "liczba złożona";
            return 0;
            
            }
        
        
        }
    
    
	cout << "Liczba pierwsza";
	return 0;
}

